core/templates/dev/head/forms/formBuilder.js

```diff
@@ -1358,6 +1378,17 @@ oppia.directive('schemaBasedFloatEditor', [function() {
           }
         };
 
+        if (typeof $scope.uiConfig() !== 'undefined' &&
+            $scope.uiConfig().slider === true) {
+          if (!$scope.localValue) {
+            $scope.localValue = 300.0;
```

Comment:
I've used `typeof $scope.uiConfig() !== 'undefined' && $scope.uiConfig().slider === true` to make sure it doesn't affect other float editors.